src/features/rewards/tweetBox/style.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const StyledTweetBox = styled<{}, 'div'>('div')`
+  border: 1px solid #d3d3d3;
```

Comment:
can you please pick color from our theme color (you can find it in storybook under `?path=/story/theme--colors`). And then you can use it like `border: 1px solid ${palette.blue400};` (replace `blue400` with your color

Reply:
Ah great, didn't know about that. Fixed.